Last-Update: 2013-01-10
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Forwarded: not-needed
Description: open .cfg file from where it is installed (not from CWD)

--- a/FreeOrion/client/human/chmain.cpp
+++ b/FreeOrion/client/human/chmain.cpp
@@ -214,9 +214,9 @@
         log_manager = new LogManager();
 
 #ifndef FREEORION_WIN32
         log_manager->createLog((GetUserDir() / "ogre.log").string(), true, false);
-        root = new Root((GetRootDataDir() / "ogre_plugins.cfg").string());
+        root = new Root("/usr/lib/freeorion/ogre_plugins.cfg");
         // this line is needed on some Linux systems which otherwise will crash with
         // errors about GLX_icon.png being missing.
         Ogre::ResourceGroupManager::getSingleton().addResourceLocation((ClientUI::ArtDir() / ".").string(),
                                                                        "FileSystem", "General");
